documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc

```diff
@@ -1,22 +1,21 @@
 [id='cli-set-topology-caches_{context}']
-= Set a Stable Topology
+= Setting a stable topology
 
 By default, after a cluster shutdown, {brandname} waits for all nodes to join the cluster and restore the topology.
-However, we offer a CLI command to mark the current topology stable for a specific cache.
+However, you can mark the current topology stable for a specific cache by using the `topology set-stable` CLI command.
 
 .Procedure
 
 . Create a CLI connection to {brandname}.
-. Do one of the following:
-+
-* Set the current topology as stable for the given cache.
+
+. Set the current topology as stable for the given cache.
 +
 [source,options="nowrap",subs=attributes+]
 ----
 topology set-stable cacheName
 ----
 +
-* If the current topology is missing more nodes than the number of owners, the force flag is necessary to confirm the operation.
+If the current topology is missing more nodes than the number of owners, the force flag is necessary to confirm the operation.
```

Comment:
it's not 'more nodes than..' but 'equal or more'.

Reply:
updated